rdm/0003-persistent-identifiers.md

```diff
@@ -0,0 +1,105 @@
+- Start Date: 2019-09-30
+- RFC PR: [#3](https://github.com/inveniosoftware/rfcs/pull/3)
+- Authors: Lars Holm Nielsen
+
+# Persistent identifiers
+
+## Summary
+
+Overall design for management of persistent identifiers in Invenio RDM.
+
+## Motivation
+
+- As a user, I want to have a DOI for my upload, so that I I can cite my research.
+- As a admin, I want to manage multiple DOI prefixes, so that I I don’t need a repository installation per prefix.
+- As a admin, I want to assign both DOIs and Handles for uploads to my repository, so that I I can fulfill business requirements.
+- As a admin, I want to have a Handle for one upload and a DOI for another upload, so that I I don’t need a repository installation per PID scheme.
+- As a user, I want to share a dataset that already have a DOI, so that I I can collect all my research one place.
+- As a admin, I want to have tombstone pages, so that I I can remove content.
+- As a admin, I want to take over management of a dataset with a DOI from another repository, so that I records can be migrated between repositories..
```

Comment:
```suggestion
- As an admin, I want to take over management of a dataset with a DOI from another repository, so that records can be migrated between repositories..
```